services/cooperation.js

```diff
@@ -32,6 +32,8 @@ const cooperationService = {
     const { role: currentUserRole } = currentUser
     const { price, status } = updateData
 
+    if (price && status) throw createForbiddenError()
```

Comment:
According to the model price has `Number` type. So, if we obtain new status and new price with 0 value `price && status` will be `false`. That's right? Therefore, this error will not be generated for this case. I see, that `updateCooperation` does not check case when `price===0`.

Reply:
Indeed, I missed it. Changed the condition ✅